doc/api/events.markdown

```diff
@@ -20,7 +20,10 @@ attached to.
 
 ## Class: events.EventEmitter
 
-To access the EventEmitter class, `require('events').EventEmitter`.
+To access the EventEmitter class, require `events`.
+```javascript
+var EventEmitter = require('events');
+```
```

Comment:
Better: 

```
Use `require('events')` to access the EventEmitter class.
```

This would be more inline with the general formatting themes in lots of other modules' docs.